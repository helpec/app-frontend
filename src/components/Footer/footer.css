footer {
  background: #762386;
  color: #fff;
}

footer a {
  color: #fff;
}

footer h1, footer h2, footer h3, footer h4, footer h5, footer h6 {
  color: fff;
}

footer address {
  line-height: 1.6em;
}

footer h3 a:hover, footer a:hover {
  text-decoration: none;
  margin: 0;
  padding: 0;
}

ul.social-network {
  list-style: none;
  margin: 0;
}

ul.social-network li {
  display: inline;
}

#sub-footer {
  background: #0a0a0a;
  text-shadow: none;
  color: #5d5d5d;
  padding: 0;
  padding-top: 0px;
}

#sub-footer p {
  margin: 0;
  padding: 0;
}

#sub-footer span {
  color: #c7c7c7;
}

.copyright {
  text-align: center;
  font-size: 12px;
  color: #fff;
}

#sub-footer ul.social-network {
  float: right;
}

/* ===============================
 Live preview purpose
================================ */

/* --- buttons --- */

.bs-docs-example {
  margin: 0 0 10px 0;
}

.demobtn a.btn {
  margin: 0 10px 10px 0;
}

section.demogrid .span1, section.demogrid .span2, section.demogrid .span3, section.demogrid .span4, section.demogrid .span5, section.demogrid .span6, section.demogrid .span7, section.demogrid .span8, section.demogrid .span9, section.demogrid .span10, section.demogrid .span11 {
  background: #f6f6f6;
  text-align: center;
  padding: 20px 0 20px;
}

section.demogrid .show-grid .row, section.demogrid .show-grid .row-fluid {
  margin-bottom: 0;
}

section.demogrid .show-grid .nest {
  background: #ddd;
  text-align: center;
  padding: 20px 0 20px;
}

section.demogrid .span12.nest {
  background: #f6f6f6;
  text-align: center;
  padding: 20px 0 0;
}

.demoinline p {
  display: inline;
  margin-right: 20px;
}

/* --- demo animations --- */

.demo a.btn {
  margin: 0 10px 10px 0;
}
