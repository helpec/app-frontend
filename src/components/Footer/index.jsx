import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import './footer.css'

class Footer extends Component {
  render() {
    return (
    <footer>
      <div className="container">
        <div className="row">
          <div className="span4">
            <div className="widget">
              <h3 className="widgetheading">Nossas PÃ¡ginas</h3>
                <ul className="link-list">
                <li><Link to="/quemsomos">Quem somos</Link></li>
                <li><Link to="/falandosobre">Falando Sobre</Link></li>
                <li><Link to="/socorros">Primeiro Socorros</Link></li>
              </ul>
            </div>
          </div>
           <div className="span4">
            <div className="widget">
              <h3 className="widgetheading">Fale conosco</h3>
              <p>
                <i className="icon-phone"></i> (11) 98232-6391 <br/>
                <i className="icon-envelope-alt"></i> camilam.oliveiralima@gmail.com
              </p>
              </div>
            </div>
              <div className="span4">
              <div className="widget">
             <h3 className="widgetheading">Redes Sociais</h3>
              <ul className="social-network">
                <li><a href="https://www.facebook.com/" data-placement="bottom" title="Facebook">
                  <i className="icon-facebook icon-square"></i></a></li>
                <li><a href="https://www.linkedin.com/in/camilamlima/" data-placement="bottom" title="Linkedin">
                  <i className="icon-linkedin icon-square"></i></a></li>
                 <span>&Camila Lima - Todos os direitos reservados.</span>
              </ul>
               </div>
               </div>
          </div>
        </div>

      </footer>
    );
  }
}

export default Footer;