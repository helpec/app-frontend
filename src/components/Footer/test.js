import React from 'react';
import { BrowserRouter } from "react-router-dom";
import renderer from 'react-test-renderer';

import Footer from '.';

test('Testar componente de Footer da aplicação', () => {

    const component = renderer.create(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
