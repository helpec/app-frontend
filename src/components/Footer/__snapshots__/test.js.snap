// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar componente de Footer da aplicação 1`] = `
<footer>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="span4"
      >
        <div
          className="widget"
        >
          <h3
            className="widgetheading"
          >
            Nossas Páginas
          </h3>
          <ul
            className="link-list"
          >
            <li>
              <a
                href="/quemsomos"
                onClick={[Function]}
              >
                Quem somos
              </a>
            </li>
            <li>
              <a
                href="/falandosobre"
                onClick={[Function]}
              >
                Falando Sobre
              </a>
            </li>
            <li>
              <a
                href="/socorros"
                onClick={[Function]}
              >
                Primeiro Socorros
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="span4"
      >
        <div
          className="widget"
        >
          <h3
            className="widgetheading"
          >
            Fale conosco
          </h3>
          <p>
            <i
              className="icon-phone"
            />
             (11) 98232-6391 
            <br />
            <i
              className="icon-envelope-alt"
            />
             camilam.oliveiralima@gmail.com
          </p>
        </div>
      </div>
      <div
        className="span4"
      >
        <div
          className="widget"
        >
          <h3
            className="widgetheading"
          >
            Redes Sociais
          </h3>
          <ul
            className="social-network"
          >
            <li>
              <a
                data-placement="bottom"
                href="https://www.facebook.com/"
                title="Facebook"
              >
                <i
                  className="icon-facebook icon-square"
                />
              </a>
            </li>
            <li>
              <a
                data-placement="bottom"
                href="https://www.linkedin.com/in/camilamlima/"
                title="Linkedin"
              >
                <i
                  className="icon-linkedin icon-square"
                />
              </a>
            </li>
            <span>
              &Camila Lima - Todos os direitos reservados.
            </span>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
